sdk/src/test/java/io/opentelemetry/sdk/resources/EnvVarResourceTest.java

```diff
@@ -67,4 +67,15 @@ public void parseResourceAttributes_quotes() {
     Attributes result = EnvVarResource.parseResourceAttributes("value=\"foo\"");
     assertThat(result).isEqualTo(Attributes.of("value", stringAttributeValue("foo")));
   }
+
+  @Test
+  public void getResourceAttributes_properties() {
```

Comment:
Please, add also a test for environment variables. An example is available here: https://github.com/open-telemetry/opentelemetry-java/blob/46f0c5f0b7b5044029fb55b105bf00576665316b/sdk/src/test/java/io/opentelemetry/sdk/common/export/ConfigBuilderTest.java#L270

Reply:
fixed at lastest commit.

Reply:
Thank you :)